service/integration/test/test_batch_perf.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+#  Copyright (c) 2015 - 2024, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+import matplotlib.pyplot as plt
+
+def main():
+    cases = ['msr-safe-root',
+             'msr-safe-service',
+             'msr-sync-root',
+             'msr-sync-service',
+             'msr-uring-root',
+             'msr-uring-service']
+    time_samples = dict()
+    for cc in cases:
+        time_samples[cc] = []
+        with open(f'{cc}.csv') as fid:
+            fid.readline()
+            for line in fid.readlines():
+                (count, time) = line.split(',')
+                time_samples[cc].append(float(time))
+
+    height = [sum(vv)/len(vv) for vv in time_samples.values()]
```

Comment:
Some of the cases spend more time on the first sample than on the rest of the samples (e.g., due to obtaining a session). Can we add error bars and/or separate bars for first&rest iterations so that information isn't lost in the mean delay?